frontend/javascripts/dashboard/folders/details_sidebar.tsx

```diff
@@ -79,6 +84,26 @@ function getMaybeSelectMessage(datasetCount: number) {
 function DatasetDetails({ selectedDataset }: { selectedDataset: APIDatasetCompact }) {
   const context = useDatasetCollectionContext();
   const { data: fullDataset, isFetching } = useDatasetQuery(selectedDataset);
+  const activeUser = useSelector((state: OxalisState) => state.activeUser);
+  const owningOrganizationDisplayName = useFetch(
+    async () => {
+      const owningOrga = await getOrganization(selectedDataset.owningOrganization);
+      return owningOrga.displayName;
+    },
+    null,
+    [],
```

Comment:
if we would stick to useFetch, this would need to be `[selectedDataset.owningOrganization]`. this means that the fetch will be re-executed as soon as the owning organization changes. this would happen when you first select a dataset from orga A and then a dataset from orga B. since the sidebar is not re-mounted, only the selected dataset changes. currently, the sidebar would show the wrong organization for that case. you can check this by opening the network tab in the devtools and selecting different datasets (only the first selection will trigger a request, which is nice on the one hand because "caching", but on the other hand it will never re-fetch even when it's necessary ^^).

this means that my comment from above ("this request will always be triggered now when the the sidebar changes because the user clicked on a different dataset.") is not correct. however, I would still recommend the usage of react-query, since it will fix the bug and ensure caching etc.